### PR TITLE
Judge if a retrieved release is nil and update version

### PR DESCRIPTION
**Pull request by EmilyEmily**

### CODE REVIEW

File: updatecfg.go [hunk 1]
```diff
@@ -168,6 +169,10 @@ func (cmd *updateConfigCommand) run() error {
 		return err
 	}
 
+	if reflect.ValueOf(ls).IsNil() {
```
Comment (mattcui) [inline updatecfg.go]: Just curious whether it returned the empty list of `ls.Releases` or `ls` is nil?

Comment (zhanggbj) [reply updatecfg.go]: Please removed commented code if not needed any more, thanks!

File: updatecfg.go [hunk 2]
```diff
@@ -168,6 +169,10 @@ func (cmd *updateConfigCommand) run() error {
...
+		return errors.New("the releases from helm list is empty")
```
Comment (zhanggbj) [inline updatecfg.go]: I think If there's an error here, it means we cannot find the release named xxxx, but not helm list is empty, right?

**Review by zhanggbj — CHANGES_REQUESTED** [review]

File: updatecfg.go (hunk 1 above)
Comment (EmilyEmily) [reply updatecfg.go]: > Just curious whether it returned the empty list of `ls.Releases` or `ls` is nil?

ls is nil, so ls.Releases will panic.

Comment (EmilyEmily) [reply updatecfg.go]: > Please removed commented code if not needed any more, thanks!

Done

File: updatecfg.go (hunk 2 above)
Comment (EmilyEmily) [reply updatecfg.go]: Right, updated.

**Review by zhanggbj — APPROVED** [review]

**Review by zhanggbj — APPROVED** [review]

**Pull request merged**